spec/integration/api_resources_controller_spec.rb

```diff
@@ -38,4 +38,58 @@
       end
     end
   end
+
+  path '/api/learning_units/{learning_unit_id}/resources' do
+    post 'Creates a new resource for the given learning unit' do
+      tags 'Resources'
+      consumes 'application/json'
+      operationId 'createResource'
+      produces 'application/json'
+      parameter name: :learning_unit_id, in: :path, type: :string
+      parameter name: :resource, in: :body, schema: {
+        type: :object,
+        properties: {
+          name: { type: :string },
+          url: { type: :string }
+        }
+      }
+
+      response '201', 'Created' do
+        schema type: :object, properties: {
+          id: { type: :integer },
+          name: { type: :string },
+          url: { type: :string }
+        }
+        let(:learning_unit_id) { create(:learning_unit).id }
+        let(:resource) { { name: 'test_resource', url: 'http://test.io' } }
+        run_test!
+      end
+
+      response '400', 'Invalid parameters' do
```

Comment:
if the URL's prefix is the only parameter that we are validating, I think that we need to be clearer on what's the reason of the error. At least on the documentation that we're creating.

I tried the API on a local environment via swagger and it was not intuitive that the default parameters (`{
  "name": "string",
  "url": "string"
}`) did not work properly and returned 400

Reply:
done